src/lwip/qpg6100/lwipopts.h

```diff
@@ -89,11 +89,11 @@
 #define MEMP_SEPARATE_POOLS (1)
 #define LWIP_PBUF_FROM_CUSTOM_POOLS (0)
 #define MEMP_USE_CUSTOM_POOLS (0)
-#define PBUF_POOL_SIZE (8)
+#define PBUF_POOL_SIZE (4)
```

Comment:
This does not seem obviously related to the rest of these changes...

Reply:
This is to pass CI since we reached the memory limit on qpg6100, this change has already been merged with https://github.com/project-chip/connectedhomeip/pull/5610.

Reply:
Great, thank you for spinning that off into a separate PR!